frontend/src/components/header/Header.tsx

```diff
@@ -309,16 +315,19 @@ export default function Header({
 
   const getLogo = () => {
     let imageUrl = "/images";
-    if (!isCustomHub) {
-      if (isHubPage && isLocationHub) {
+    if (isCustomHub) {
+      imageUrl = `/images/hub_logos/prio1.png`;
+    } else if (isHubPage) {
+      if (hubUrl && isLandingPage) {
+        imageUrl += `/hub_logos/ch_${hubUrl.toLowerCase()}_logo_white.svg`;
+      } else if (isLocationHub) {
         imageUrl += `/hub_logos/ch_${hubUrl}_logo.svg`;
       } else {
         imageUrl = loadDefaultLogo(transparentHeader, isMediumScreen);
       }
     } else {
```

Comment:
This should achieve the same, right? If yes, this is simpler.

```suggestion
    } else if (isLocationHub) {
      if (hubUrl && isLandingPage) {
        imageUrl += `/hub_logos/ch_${hubUrl.toLowerCase()}_logo_white.svg`;
      } else {
        imageUrl += `/hub_logos/ch_${hubUrl}_logo.svg`;
      }
    } else {
```